### PR TITLE
[webnn] Add float32 tests for element-wise unary identity operation

### PR DESCRIPTION
**Pull request by BruceDai**
@fdwr @huningxin @Honry PTAL, thanks.

### CODE REVIEW

File: webnn/resources/test_data/copy.json [hunk 1]
```diff
@@ -0,0 +1,348 @@
...
+          -2.5986480712890625,
+          -7.264847278594971,
+          -9.782458305358887,
+          5.496699810028076,
+          -9.967339515686035,
+          -6.901016712188721,
+          -2.8501904010772705,
+          3.279616355895996
+        ],
+        "type": "float32"
```
Comment (fdwr) [inline webnn/resources/test_data/copy.json:344]: How about a float16 case? (I forget if there were any blocking reasons why not)

I reduced the numbers to float16 precision here;

```
 0.37744140625,
-0.88916015625,
 9.96875,
 7.96484375,
-4.20703125,
-3.748046875,
-2.51171875,
 5.77734375,
-4.47265625,
-8.40625,
-3.857421875,
 6.03515625,
 9.078125,
 7.1015625,
 1.4169921875,
-5.64453125,
-2.599609375,
-7.265625,
-9.78125,
 5.49609375,
-9.96875,
-6.90234375,
-2.849609375,
 3.279296875,
```

**Review by fdwr — APPROVED** [review]
Bruce, thanks for implementing identity.

File: webnn/resources/test_data/copy.json (hunk 1 above)
Comment (BruceDai) [reply webnn/resources/test_data/copy.json:344]: >  How about a float16 case?

I will submit another PR for only float16 tests. 

And if you agree with my current solution of testing float16 tests on https://github.com/web-platform-tests/wpt/pull/42420, I will add such float16 tests soon.

Comment (fdwr) [reply webnn/resources/test_data/copy.json:344]: Okie dokie.

**Review by Honry — APPROVED** [review]
LGTM, thanks!

**Comment by BruceDai** [comment]
Hi @fdwr, I synced with @huningxin, we decided to rename **copy** tests to previous **identity** tests, since Chromium WebNN API implementation has already supported **identity** op, then these WPT **identity** tests can be ran.

**Review by fdwr — APPROVED** [review]

**Review by Honry — APPROVED** [review]
LGTM, thanks!

**Pull request merged**